sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/ExpressionComplexity.cs

```diff
@@ -17,6 +17,10 @@ public bool M()
 
             var d1 = true && false && true && false && true && true; // Noncompliant
 
+            bool? v1 = null, v2 = null, v3 = null, v4 = null, v5 = null;
```

Comment:
should you add tests to CognitiveComplexity.cs and FunctionComplexity.cs as well?